emails/utils.py

```diff
@@ -220,3 +214,50 @@ def generate_relay_From(original_from_address):
         )
     )
     return formatted_from_address
+
+
+def get_message_id_bytes(message_id_str):
+    message_id = message_id_str.split("@", 1)[0].rsplit("<", 1)[-1].strip()
+    return message_id.encode()
+
+
+def b64_lookup_key(lookup_key):
+    return base64.urlsafe_b64encode(lookup_key).decode('ascii')
```

Comment:
look into seeing if we can use the `utf-8`. The encode only returns the `ascii` strings only.

Reply:
cursory search showed that urlsafe_b64encode uses binary data or `ascii` string so we should continue to use that format. See this [doc](https://docs.python.org/3/library/base64.html) for more info.